Nutraceuticals Outline and Checklist (#748)
greenelab/covid19-review: content/21.nutraceuticals.md

Considerable interest amongst the scientific community and emerging evidence has led to further investigation of these products. 
In this review, we critically appraise the evidence surrounding dietary supplements and nutraceuticals for the prophylaxis and treatment of COVID-19.
These include vitamin C, vitamin D, omega 3 polyunsaturated fatty acids (PUFA), probiotics, and zinc.
These were selected as they are currently under clinical investigation.
There is increasing evidence to suggest that vitamin D supplementation may be beneficial due to plausible associations between vitamin D deficiency with COVID-19 incidence and possibly disease severity.
A growing number of scientific organizations are considering or advising for increased vitamin D supplementation to those at high risk of COVID-19.
Overall, caution is warranted as further evidence and clinical trials are required before conclusive evidence-based recommendations can be formulated for any specific dietary supplements or nutraceuticals.
Consumers should also be aware of misinformation and false promises surrounding some supplement products, which may lack safety and efficacy evidence due to poor regulation by authorities.

> However, considering that nutritional status plays a significant role in patient outcomes, it would be pertinent to advise the general public to follow a healthy diet and lifestyle to prevent nutrient deficiencies and to maintain a healthy immune system.

Author: ```suggestion
However, considering that nutritional status plays a significant role in patient outcomes, it would be prudent to advise the general public to follow a healthy diet and lifestyle to prevent nutrient deficiencies and to maintain a healthy immune system.
```

However, little is known about the efficacy of these dietary supplements and nutraceuticals against the novel coronavirus (SARS-CoV-2) or the disease it causes, COVID-19.
This review provides a critical overview of the potential prophylactic and therapeutic value of various dietary supplements and nutraceuticals from the evidence available to date.
Evidence indicates that vitamin D deficiency may be associated with greater incidence of infection and severe COVID-19 and that vitamin D supplementation may be of prophylactic and therapeutic value.
However, considerably more research is required to determine whether dietary supplements and nutraceuticals exhibit prophylactic and therapeutic value against SARS-CoV-2 infection and COVID-19. 

### Introduction

Scientists and the medical community are scrambling to repurpose or discover novel host-directed therapies as a result of the coronavirus diseases 2019 (COVID-19) pandemic caused by the spread of the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2).
Some successful therapies such as remdesivir and dexamethasone have been identified for hospitalized patients.

> Furthermore, society has mostly adopted the non-pharmacological preventative public health strategies that reduce the transmission of SARS-CoV-2. 

Author: HaHa true! 
```suggestion
Furthermore, most societies have adopted the non-pharmacological preventative public health strategies that reduce the transmission of SARS-CoV-2. 
```

However, many have sought additional protections via the consumption of various dietary supplements and nutraceuticals with purported beneficial effects.
Indeed, a patients nutritional status seems to play a role in COVID-19 susceptibility and outcomes [@doi:10.3390/nu12051466; @doi:10.1101/2020.10.19.20214395; @doi:10.1093/advances/nmaa125; @doi:10.1016/j.ijid.2020.08.018; @doi:10.3390/nu12041181].

> Nutraceuticals are classified as supplements with health benefits beyond their basic nutritional value [@doi:10.1080/10408390902841529; @doi:10.1038/nrcardio.2016.103].

Author: That was perfect, nicely put

The key difference between a dietary supplement and a nutraceutical is that nutraceuticals should not only supplement the diet, but also aid in the prophylaxis and/or treatment of a disorder or disease [@doi:10.1208/ps050325].

> For the general public in particular, whether nutraceuticals or dietary supplements can provide any prophylactic or therapeutic benefit has been a topic of interest.

Author: Haha! That was spot on.

Despite the lack of any evidence to support their use, the begining of the pandemic led sales of these products to soar. 

> In the United States, dietary supplement and nutraceutical sales were worth $345 million for the entirety of 2019, whereas sales reached $435 million during the six-week period of the outbreak that ended on April 5 of 2020, which grew by a further $151 million by May 17 2020 [@url:https://www.nutritionaloutlook.com/view/peak-dietary-supplement-sales-leveling-during-covid-19-pandemic-growth-still-remains-strong].

Author: It’s crazy how word of mouth, old adages, marketing, and general perceptions can drive sales of products into the millions of $$$ 

And the fact that research here is so scarce is really fascinating. The problem is funding is so poor for this research. If it is not a pharmaceutical it is not worth funding is the general mantra.

These regulatory disparities have been highlighted by the pandemic as many individuals and companies have attempted to profit from the vulnerabilities of others by overstating claims in relation to the treatment of COVID-19 using supplements and nutraceuticals. 
The FDA has written several letters to prevent companies marketing or selling products based on false hyperbolic promises to prevent SARS-CoV-2 infection or treat COVID-19 [@url:https://www.fda.gov/inspections-compliance-enforcement-and-criminal-investigations/warning-letters/noetic-nutraceuticals-607572-05152020; @url:https://beta.regulations.gov/document/FDA-2020-S-0023-0068; @url:https://www.fda.gov/inspections-compliance-enforcement-and-criminal-investigations/warning-letters/spartan-enterprises-inc-dba-watershed-wellness-center-610876-10302020], some even charging the consumers $23,000 [@url:https://www.ftc.gov/news-events/press-releases/2020/07/ftc-sues-california-marketer-23000-covid-19-treatment-plan].

> There has even been some incidences highlighted in the media because of their potentially life threatening consequences, for example the use of oleandrin was touted as a potential 'cure' by individuals close to the former President of the United States despite its high toxicity [@url:https://cen.acs.org/biological-chemistry/natural-products/oleandrin-compound-touted-possible-COVID/98/web/2020/08].

Author: @rando2, good idea. See if this makes sense.

> There has even been some incidences highlighted in the media because of their potentially life threatening consequences, for example the use of oleandrin was touted as a potential 'cure' by individuals close to the former President of the United States despite its high toxicity [@url:https://cen.acs.org/biological-chemistry/natural-products/oleandrin-compound-touted-possible-COVID/98/web/2020/08].

Reviewer: Looks good to me!

A host of potential candidates have been highlighted in the literature that target various aspects of the COVID-19 viral pathology, while others are thought to prime the host immune system.
These candidates include vitamins and minerals along with extracts and omega-3 polyunsaturated fatty acids (n-3 PUFA) [@doi:10.1002/jmv.25707].
_In vitro_ and _in vivo_ studies suggest that nutraceuticals containing phycocyanobilin, N-acetylcysteine, glucosamine, selenium or phase 2 inductive nutraceuticals (e.g. ferulic acid, lipoic acid, or sulforaphane) can prevent or modulate RNA virus infections via amplification of the signaling activity of mitochondrial antiviral-signaling protein (MAVS) and activation of toll-like receptor 7 (TLR7) [@doi:10.1016/j.pcad.2020.02.007].
While promising, further animal and human studies are required to assess the therapeutic potential of these various nutrients and nutraceuticals against COVID-19.
For the purpose of this review, we have highlighted some of the main dietary supplements and nutraceuticals that are currently under investigation for their potential prophylactic and therapeutic applications.

> These include vitamins C and D, zinc, probiotics, and n-3 PUFA. 

Author: Done haha!

An observational study has shown that COVID-19 patients had significantly lower zinc levels in comparison to healthy controls, they also tended to have more complications (70.4% vs 30.0%, _p_ = 0.009), and potentially prolonged hospital stays (7.9 vs 5.7 days, _p_ = 0.048) [@doi:10.1016/j.ijid.2020.09.014].  
In coronaviruses specifically, _in vitro_ evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP (a fluorescently tagged SARS-CoV-1) and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015].
Currently, there are over twenty clinical trials registered with the intention to use zinc in a preventative or therapeutic manner.
However, many of these trials proposed the use of zinc in conjunction with hydroxychloroquine and azithromycin [@clinicaltrials:NCT04370782; @clinicaltrials:NCT04377646; @clinicaltrials:NCT04395768; @clinicaltrials:NCT04373733], and it is not known how the lack of evidence supporting the use of hydroxychloroquine will affect investigation of zinc.

> One retrospective observational study of NYU Lagone hospitals in New York compared outcomes among hospitalized COVID-19 patients administered hydroxychloroquine and azithromycin with zinc sulphate (n = 411) versus hydroxychloroquine and azithromycin alone (n = 521).

Author: Added a line, if you want to check it out.

> One retrospective observational study of NYU Lagone hospitals in New York compared outcomes among hospitalized COVID-19 patients administered hydroxychloroquine and azithromycin with zinc sulphate (n = 411) versus hydroxychloroquine and azithromycin alone (n = 521).

Reviewer: Looks great!

Vitamin C is utilized by the body during viral infections, as evinced by lower concentrations in leukocytes and lower concentrations of urinary vitamin C.
Post-infection, these levels return to baseline ranges [@doi:10.1177/003693307301800102; @doi:10.1111/j.1749-6632.1975.tb29312.x; @doi:10/fd22sv; @doi:10.3390/nu9040339; @doi:10.1079/bjn19920004].
Indeed, COVID-19 patients appear to be deficient in vitamin C [@doi:10.1016/j.medidd.2020.100064; @doi:10.1186/s13054-020-03249-y]. 
COVID-19 is also associated with the formation of microthrombi and coagulopathy [@doi:10.1172/JCI140183] that contributes to the observed lung pathology [@doi:10.1183/16000617.0240-2020], which can be ameliorated by early infusions of vitamin C by inhibiting endothelial surface P-selectin expression and platelet-endothelial adhesion [@doi:10.3390/antiox6030049].

> Indeed, intravenous vitamin C also reduced D-dimer levels in a case study of 17 COVID-19 patients [@doi:10.1080/14787210.2020.1794819].

Author: I’ve added one or two of them.

These studies are supported by several others that indicate that vitamin D status may be an independent risk factor for the severity of COVID-19 [@doi:10.1371/journal.pone.0239799; @doi:10.1016/j.jiph.2020.06.021; @doi:10.1111/febs.15495; @doi:10.1001/jamanetworkopen.2020.19722] and in COVID-19 patients relative to population-based controls [@doi:10.1210/clinem/dgaa733].
Indeed, serum concentrations of 25-hydroxyvitamin D above 30 ng/ml, which indicate vitamin D sufficiency, seems to be associated with a reduction in serum C-reactive protein (CRP), an inflammatory marker, along with increased lymphocyte levels, which suggests that vitamin D levels may modulate the immune response by reducing risk for cytokine storm in response to SARS-CoV-2 infection [@doi:10.1210/clinem/dgaa733].
A study in India has determined that COVID-19 fatality is higher in patients with severe COVID-19 and low serum 25-hydroxyvitamin D (mean level 6.2 ng/ml; 97% vitamin D deficient) levels versus asymptomatic non-severe patients with higher levels of vitamin D (mean level 27.9 ng/ml; 33% vitamin D deficient) [@doi:10.1038/s41598-020-77093-z]. 

> In the same study, vitamin D deficiency was associated with higher levels of inflammatory markers including IL-6, ferritin, and TNFa.

Author: I just didn’t know how to do it! I use alpha too.

Overall, the evidence suggest that there is likely a link between low serum 25-hydroxyvitamin D and COVID-19 incidence, these studies must be interpreted with caution as there is the potential for reverse causality, bias, and other confounding factors including that vitamin D deficiency is also associated with numerous pre-existing conditions and risk factors that can increase the risk for severe COVID-19 [@doi:10.3390/nu12051466; @doi:10.1038/s41371-020-00398-z; @doi:10.1111/joim.13149; @doi:10.1007/s40520-020-01677-y].   

While these studies inform us of the potential importance of vitamin D sufficiency and the risk of SARS-CoV-2 infection and severe COVID-19, they fail to conclusively determine whether vitamin D supplementation can therapeutically affect the clinical course of COVID-19.
In one study, 40 vitamin D deficient asymptomatic or mildly symptomatic participants patients were either randomized to receive 60,000 IU of cholecalciferol daily for at least 7 days (n = 16) or a placebo (n = 24) with a target serum 25-hydroxyvitamin D level >50 ng/ml.
At day 7, 10 patients achieved >50 ng/ml, and another 2 by day 14.

> By the end of the study, the treatment group had a greater proportion of vitamin D-deficient participants that turned SARS-CoV-2 RNA negative and they had a significantly lower fibrinogen levels potentially indicating a beneficial effect [@doi:10.1136/postgradmedj-2020-139065].

Author: So I actually found the phrasing in the paper confusing myself. I had wrote tested negative too and changed it because they are insinuating that the vitamin D presence may have been a contributing factor. I think its just a complicated result to explain. 

To answer your question. The treatment group and control group were sometimes discharged from hospital before testing negative because they were only mildly affected. The authors are insinuating that those who were treated with vitamin D tested negative for SARS-CoV-2 in greater proportions than those who did not receive vitamin D by the end of the study or before being discharged. Does that make sense? I only included this study because it is the only study that kind of touches upon the supposed antiviral effects of vitamin D, but the unfortunate thing is that it is a rushed study and with a poor methodology.

A pilot study in Spain determined that early high dose calcifediol (~21,000 IU days 1-2 and ~11,000 IU days 3-7 of hospital admission) with hydroxychloroquine and azithromycin to 50 hospitalized COVID-19 patients significantly reduced ICU admissions and may have reduced disease severity versus hydroxychloroquine and azithromycin alone [@doi:10.1016/j.jsbmb.2020.105751].  
While this study received significant criticism the National Institute for Health and Care Excellence (NICE) in the UK [@url:https://www.nice.org.uk/advice/es28/chapter/Key-messages], an independent follow-up statistical analysis supported the findings of the study as a result of cholecalciferol treatment [@doi:10.1101/2020.11.08.20222638]. 
Another trial of 986 COVID-19 hospitalized patients in three UK hospitals, administered cholecalciferol supplementation (≥ 280,000 IU in a time period of 7 weeks) to 151 patients, which was associated with a reduced risk of COVID-19 mortality regardless of baseline 25-hydroxyvitamin D levels [@doi:10.3390/nu12123799].

> Juxtaposed, a double-blind, randomized, placebo-controlled trial of 240 hospitalized COVID-19 patients in Sao Paulo Brazil administered a single 200,000 IU oral dose of vitamin D. 

Author: I think you’re right, went for the old faithful ‘indeed’

Probiotics and next-generation probiotics, which are more akin to pharmacological-grade supplements, have been associated with multiple potential beneficial effects for allergies, digestive tract disorders, and even metabolic diseases through their anti-inflammatory and immunomodulatory effects [@doi:10.1038/nmicrobiol.2017.57; @doi:10.1093/advances/nmy063].
However, the mechanisms by which probiotics affect these various conditions would likely differ among strains, with the ultimate effect of the probiotic depending on the heterogeneous set of bacteria present [@doi:10.1093/advances/nmy063].
Some of the beneficial effects of probiotics include reducing inflammation by promoting the expression of anti-inflammatory mediators, inhibiting toll-like receptors (TLR) 2 and 4, direct competition with pathogens, the synthesis of antimicrobial substances or other metabolites, improving intestinal barrier function, and/or favorably altering the gut microbiota and the brain-gut axis [@doi:10.1016/j.earlhumdev.2019.05.010; @doi:10.1093/advances/nmy063; @doi:10.1159/000342079].

> It is also thought that lactobacilli such as _Lactobacillus paracasei, Lactobacillus plantarum_ and _Lactobacillus rhamnosus_ have the capacity to bind and inactivate some viruses via adsoptive and/or trapping mechanisms [@doi:10.1016/j.ijfoodmicro.2006.10.044].

Reviewer: ```suggestion
It is also thought that lactobacilli such as _Lactobacillus paracasei, Lactobacillus plantarum_ and _Lactobacillus rhamnosus_ have the capacity to bind to and inactivate some viruses via adsorptive and/or trapping mechanisms [@doi:10.1016/j.ijfoodmicro.2006.10.044].
```
Just to check, is it adsoptive or adsorptive?

> Other probiotic lactobacilli and even non-viable bacterium-like particles have been shown to reduce viral attachment to host cells and reduce viral titers, reduce cytokine synthesis, enhance the antiviral IFN-α response along with inducing various other antiviral mechanisms  [@doi:10.1111/lam.12160; @doi:10.1016/j.anaerobe.2011.04.022; @doi:10.4049/jimmunol.1800927; @doi:10.1016/j.ijfoodmicro.2006.10.044; @doi:10.3390/medicina55090519; @doi:10.1038/s41598-019-39602-7; @doi:10.3920/bm2016.0160; @doi:10.3389%2Ffimmu.2020.00015; @doi:10.1016/j.intimp.2013.12.020; @doi:10.1016/j.intimp.2019.106115].

Reviewer: ```suggestion
Other probiotic lactobacilli and even non-viable bacterium-like particles have been shown to reduce both viral attachment to host cells and viral titers, along with reducing cytokine synthesis, enhancing the antiviral IFN-α response, and inducing various other antiviral mechanisms  [@doi:10.1111/lam.12160; @doi:10.1016/j.anaerobe.2011.04.022; @doi:10.4049/jimmunol.1800927; @doi:10.1016/j.ijfoodmicro.2006.10.044; @doi:10.3390/medicina55090519; @doi:10.1038/s41598-019-39602-7; @doi:10.3920/bm2016.0160; @doi:10.3389%2Ffimmu.2020.00015; @doi:10.1016/j.intimp.2013.12.020; @doi:10.1016/j.intimp.2019.106115].
```
Just an idea since there were a bunch of "reduce"s in a row!

> These antiviral and immunobiotic mechanisms and more have been reviewed in detail elsewhere [@doi:10.3390/nu12061718; @doi:10.1038/s41538-020-00078-9; @doi:10.3389/fpubh.2020.00186]. 

Reviewer: ```suggestion
These antiviral and immunobiotic mechanisms and others have been reviewed in detail elsewhere [@doi:10.3390/nu12061718; @doi:10.1038/s41538-020-00078-9; @doi:10.3389/fpubh.2020.00186]. 
```

Consequently, several investigations are underway to investigate the prophylactic and therapeutic potential of probiotics for COVID-19.
However, blind use of conventional probiotics for COVID-19 is cautioned against until the pathogenesis of SARS-CoV-2 is further established [@doi:10/d2qq].
Until clinical trials investigating the prophylactic and therapeutic potential of probiotics for COVID-19 are complete, it is not possible to provide an evidence-based recommendation for their use.
Despite these concerns, complementary use of probiotics as an adjuvant therapeutic has been proposed by the Chinese National Health Commission and National Administration of Traditional Chinese Medicine [@doi:10.1111/1751-2980.12851]. 

### Discussion

The purported prophylactic and therapeutic benefits of dietary supplements and nutraceuticals for multiple disorders, diseases, and infections has been the subject of significant research and debate for the last few decades. 

> Inevitably, scientists are also investigating the potential of these various products to treat or prevent COVID-19.

Reviewer: ```suggestion
Inevitably, scientists are also investigating the potential for these various products to treat or prevent COVID-19.
```

This interest also extends to consumers, which led to a remarkable increase of sales of dietary supplements and nutraceuticals throughout the pandemic due to a desire to obtain additional protections from infection and disease.
It is pertinent to clarify that the nutraceuticals discussed in this review, namely vitamin C, vitamin D, n-3 PUFA, zinc, and probiotics, have been selected because of their possible relevance to the biological mechanisms that can beneficially affect viral and respiratory infections and because they are currently under clinical investigation.

> However, there are various other products and molecules that have garnered scientific interest and are under further investigation. 

Reviewer: ```suggestion
However, there are various other products and molecules that have garnered scientific interest and require further investigation. 
```

These include polyphenols, lipid extracts, and tomato-based nutraceuticals among others, which have all been suggested for the prevention of potential cardiovascular complications of COVID-19 such as thrombosis [@doi:10.3389/fnut.2020.583080; @doi:10.3390/nu12061718].

> These are just some of the many substances and supplements under investigation that currently have limited evidence to support their use or supplementation with the aim to prevent or treat COVID-19.

Reviewer: ```suggestion
```
move down?

> Melatonin is another supplement that has been identified as a potential antiviral agent against SARS-CoV-2 using computational methods [@doi:10.1038/s41421-020-0153-3] and it has also been highlighted as a potential therapeutic agent for COVID-19 due to its documented antioxidant, anti-apoptotic, immunomodulatory, and anti-inflammatory effects [@doi:10.3389/fnut.2020.583080; @doi:10.3390/antiox10010047; @doi:10.3390/nu13010047].

Reviewer: ```suggestion
Melatonin is another supplement that has been identified as a potential antiviral agent against SARS-CoV-2 using computational methods [@doi:10.1038/s41421-020-0153-3], and it has also been highlighted as a potential therapeutic agent for COVID-19 due to its documented antioxidant, anti-apoptotic, immunomodulatory, and anti-inflammatory effects [@doi:10.3389/fnut.2020.583080; @doi:10.3390/antiox10010047; @doi:10.3390/nu13010047].
```

Notably, melatonin, vitamin D and zinc were included in the treatment plan of the Former President of the United States upon hospitalization due to COVID-19 [@doi:10.1126/science.abf0974].

> While there is plenty speculation surrounding the use of supplements, a survey of 327,720 users of the COVID Symptom Study App found that the consumption of n-3 PUFA supplements, probiotics, multivitamins, and vitamin D was associated with a lower risk of SARS-CoV-2 infection in women but not men after adjusting for potential confounders [@doi:10.1101/2020.11.27.20239087].

Reviewer: ```suggestion
These are just some of the many substances and supplements under investigation that currently have limited evidence to support their use or supplementation with the aim to prevent or treat COVID-19.
While there is plenty speculation surrounding the use of supplements, a survey of 327,720 users of the COVID Symptom Study App found that the consumption of n-3 PUFA supplements, probiotics, multivitamins, and vitamin D was associated with a lower risk of SARS-CoV-2 infection in women but not men after adjusting for potential confounders [@doi:10.1101/2020.11.27.20239087].
```

> According to the authors, the sexual dimorphism observed may in part be because supplements may better support females due to known differences between the male and female immune systems or it could be due to behavioural and health consciousness differences between the sexes [@doi:10.1101/2020.11.27.20239087].

Reviewer: ```suggestion
According to the authors, the sexual dimorphism observed may in part be because supplements may better support females due to known differences between the male and female immune systems, or it could be due to behavioral and health consciousness differences between the sexes [@doi:10.1101/2020.11.27.20239087].
```

Certainly, randomized controlled trials are required to investigate these findings further.

In this review, we report the findings to date for several dietary supplements and nutraceuticals.
While existng evidence suggests potential benefits of n-3 PUFA and probiotic supplementation for COVID-19 treatment and prophylaxis, clinical data is lacking but trials are underway.

> Both zinc and vitamin C supplementation in hospitalized patients seem to be associated with positive outcomes, however, further clinical trials are required. 

Reviewer: ```suggestion
Both zinc and vitamin C supplementation in hospitalized patients seem to be associated with positive outcomes; however, further clinical trials are required. 
```

In any case, both vitamin C and zinc intake are part of a healthy diet and both likely presents minimal risk when supplemented for, but their potential prophylactic or therapeutic effects against COVID-19 are yet to be determined.

> On the other hand, mounting evidence from observational studies indicates that there is an association between vitamin D deficiency and COVID-19 incidence, which is supported by meta-analysis [@doi:10.1080/10408398.2020.1841090].

Reviewer: ```suggestion
On the other hand, mounting evidence from observational studies indicates that there is an association between vitamin D deficiency and COVID-19 incidence has also been supported by meta-analysis [@doi:10.1080/10408398.2020.1841090].
```

> Indeed, scientists are working to affirm these findings and to determine whether a patients serum 25-hydroxyvitamin D levels are also associated with COVID-19 severity.

Reviewer: ```suggestion
Indeed, scientists are working to confirm these findings and to determine whether a patient's serum 25-hydroxyvitamin D levels are also associated with COVID-19 severity.
```

In terms of the therapeutic potential of vitamin D, initial evidence from clinical trials are conflicting, but seem to indicate that vitamin D supplementation may reduce COVID-19 severity [@doi:10.1016/j.jsbmb.2020.105751].
The various clinical trials currently underway will be imperative to provide information on the efficacious use of vitamin D supplementation for COVID-19 prevention and/or treatment.

Finally, it is known that a patient's nutritional status affects health outcomes in various infectious diseases [@doi:10.3390/nu12041181], and COVID-19 is no different [@doi:10.1016/j.clnu.2020.03.022; @doi:10.1093/advances/nmaa125; @doi:10.1016/j.clnesp.2020.09.214]. 
Some of the main risk factors for severe COVID-19, which also happen to be linked to poor nutritional status includes obesity, hypertension, cardiovascular diseases, type II diabetes mellitus, and indeed age-related malnutrition [@doi:10.1093/advances/nmaa125; @doi:10.3390/nu12051466; @doi:10.1002/jpen.1954].
Although not the main focus of this review, it is important to consider the nutritional challenges associated with severe COVID-19 patients.

> Hospitalized COVID-19 patients tend to report an unusually high loss of appetite preceeding admission, some suffer diarrhea and gastrointestinal symptoms in hospital resulting in significantly lower food intake, and patients with poorer nutritional status were more likely to have worse outcomes and require nutrition therapy [@doi:10.1002/jpen.1953].

Reviewer: ```suggestion
Hospitalized COVID-19 patients tend to report an unusually high loss of appetite preceding admission, some suffer diarrhea and gastrointestinal symptoms that result in significantly lower food intake, and patients with poorer nutritional status were more likely to have worse outcomes and require nutrition therapy [@doi:10.1002/jpen.1953].
```

> Dysphagia also seems to be a significant problem in rehabilitating COVID-19 patients, potentially contributing to poor nutritional status [@doi:10.1038/s41430-020-0664-x]. 

Reviewer: In case it's useful information, pediatric patients also sometimes have difficulty swallowing (it seems to be more associated with MIS-C)

Author: Good shout! I will put something in here.

### Conclusions

Despite all the potential benefits of nutraceutical and dietary supplement interventions presented, currently there is a paucity of clinical evidence to support their use for the prevention or mitigation of COVID-19 infection.
Nevertheless, optimal nutritional status can prime an individual’s immune system to protect against the effects of acute respiratory viral infections by supporting normal maintenance of the immune system [@doi:10.3390/nu12041181; @doi:10.3390/nu12051466].
Nutritional strategies can also play a role in the treatment of hospitalized patients, as malnutrition is a risk to COVID-19 patients [@doi:10.1038/s41430-020-0664-x].
Overall, supplementation of vitamin C, vitamin D, and zinc may be an effective method of ensuring their adequate intake to maintain optimal immune function, which may also convey beneficial effects against viral infections due to their immunomodulatory effects.
Individuals should pay attention to their nutritional status, particulalry their intake of vitamin D, considering its deficiency tends to be widespread.

> The prevailing evidence seems to indicate an association between vitamin D deficiency with COVID-19 incidence and potentially its severity [https:10.1111/joim.13149].

Reviewer: ```suggestion
The prevailing evidence seems to indicate an association between vitamin D deficiency with COVID-19 incidence and, potentially, severity [@doi:10.1111/joim.13149].
```
